jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -224,6 +224,52 @@ YUI.add('user-profile', function() {
       this.close();
     },
 
+    /**
+      Set to a new, blank canvas with no model.
+
+      @method resetModel
+    */
+    resetModel: function() {
+      var props = this.props;
+      // Reset the connection without reconnecting, so no need to pass any
+      // params.
+      props.switchModel();
+      // XXX 2016-04-07 kadams54: the random name generator is intended to be a
+      // temporary solution until we fully implement deployment flow.
+      props.dbEnvironmentSet('name', this.generateRandomName());
```

Comment:
I think whatever value is in that input will need to eventually find its way here, to be set in the DB.